conf/META-INF/persistence.xml

```diff
@@ -33,9 +33,9 @@
             <property name="eclipselink.logging.level" value="ALL" />
 
             <property name="javax.persistence.schema-generation.database.action" value="drop-and-create"/>
-            <property name="javax.persistence.schema-generation.scripts.action" value="drop-and-create"/>
-            <property name="javax.persistence.schema-generation.scripts.create-target" value="target/sampleCreate.ddl"/>
-            <property name="javax.persistence.schema-generation.scripts.drop-target" value="target/sampleDrop.ddl"/>
+            <!--<property name="javax.persistence.schema-generation.scripts.action" value="drop-and-create"/>-->
+            <!--<property name="javax.persistence.schema-generation.scripts.create-target" value="target/sampleCreate.ddl"/>-->
+            <!--<property name="javax.persistence.schema-generation.scripts.drop-target" value="target/sampleDrop.ddl"/>-->
```

Comment:
It does still create the schema. The lines that are commented AFAIK generate schema creation scripts. These scripts are not used as part of the application.